linkbox/usrp.cpp

```diff
@@ -231,7 +231,7 @@ void CUSRP::KeyTx(int bKey)
             UsrpHeader->Reserved = 0;
 
             UsrpMetaData->TLVTag = USRP_TLV_TAG_SET_INFO;
-            UsrpMetaData->TLVLength = 13 + strlen("VK2GPU");
+            UsrpMetaData->TLVLength = 14 + strlen("VK2GPU");
```

Comment:
Yeah looks like an off by one error on my part >.< Just checked the rest of the TLV header and yeah I was short a byte.

Probably worth also changing the callsign at some point. I did have a change sitting around somewhere to make it configurable, but alas life got in the way of things in the last year so I haven't touched it in a while.